### PR TITLE
adding D&I + EEO language

### LINKED ISSUE #543: Add EEO statement to website
Use leadership-approved text (below). Add to:
- [x] bottom of /apply page
- [x] hiring FAQs: PR https://github.com/usds/website/pull/545

_Text_
U.S. Digital Service is proud to be an equal opportunity employer. We strongly encourage people of color, members of racial and ethnic minority groups, women, LGBTQI+ people, those with disabilities, and Veterans to apply. We build better products when our team represents all of America.
 
We do not discriminate based off of color, race, age, religion, sex, gender identity, sexual orientation, political affiliation, pregnancy, status as a parent, national origin, disability (physical or mental), family medical history or genetic information, political affiliation, military service, or other non-merit based factors.

### PR DESCRIPTION
**Pull request by LauraUSDS**
- Adds D&I language to `/faq`
- Adds EEO language to bottom of `/apply`

Fixes #543

### CODE REVIEW

**Comment by LauraUSDS** [comment]
@drew-usds Could you pls double-check I added the D&I question correctly to the FAQ yml? This is for https://github.com/usds/website/issues/543

Also, would love help figuring out how to delete our fake Holmes commits from this PR....

File: _content/faq.yml [hunk 1]
```diff
@@ -27,10 +27,16 @@
       a: |-
         It depends. We strive to have you go through the interview process within four weeks. Onboarding can take an additional four to six weeks depending on when you get an offer from the HR team and how long the background check takes. Your time in the onboarding process is generally out of our control at USDS.
 
-    - q: "When will I hear from the U.S. Digital Service in this process?"
+    - q: "When will I hear from USDS in this process?"
       a: |-
         You should receive an email confirmation when your application is submitted. If you are selected to move on to interviews, you will be connected with a member of the USDS Talent Team. They will be your guide throughout your interview process.
 
+    - q: "How does USDS address diversity and inclusion?"
```
Comment (drew-usds) [inline _content/faq.yml]: The EEO statement alone doesn't fully address what we do regarding diversity and inclusion. Someone could read this as the _only_ thing we do is _encourage_ people to apply. This headline should either be more specifically about EEO, or we could write another graph or two about how else we support diversity and inclusion.

Comment (LauraUSDS) [reply _content/faq.yml]: Agreed, good point. I've changed the wording of the question, and asked the team who wrote this language initially to consider making a second FAQ that is more about D&I in USDS culture / day-to-day operations.

Comment (drew-usds) [reply _content/faq.yml]: 👍

**Review by drew-usds — APPROVED** [review]

**Pull request merged**